Chapters/01-memory.qmd

```diff
@@ -23,9 +23,11 @@ In this chapter, we will talk about memory. How does Zig controls memory? What
 common tools are used? Are there any important aspect that makes memory
 different/special in Zig? You will find the answers here.
 
-Every computer needs memory. Is by having memory that computers can temporarily store
-the values/results of your calculations. Without memory, programming languages would never have
-concepts such as "variables", or "objects", to store the values that you generate.
+Computers fundamentally rely on memory to function. Is by having memory that computers
+can temporarily store the data and values generated during computations
+the values/results of your calculations. Without memory, the core concepts of
+"variables" and "objects" in programming languages would be impossible.
+

```

Comment:
I would change it to the following:

Computers fundamentally rely on memory to function.
This memory acts as a temporary storage space for the data and values generated during computations. 
Without memory, the core concepts of "variables" and "objects" in programming languages would be impossible.

Reply:
Done!